### PR TITLE
Adding Events for Subgraph

### PR DESCRIPTION
**Pull request by ssubik**

### CODE REVIEW

File: contracts/dao/staking/interfaces/IStakingEvents.sol [hunk 1]
```diff
@@ -5,12 +5,13 @@
 pragma solidity 0.8.13;
 
 interface IStakingEvents {
-    event Staked(address account, uint256 indexed streamShares, uint256 indexed nVoteToken, uint256 indexed lockId);
+    event Staked(address account, uint256 amount,uint256 indexed streamShares, uint256 indexed nVoteToken, uint256 indexed lockId, uint256 end);
```
Comment (BaldyAsh) [inline contracts/dao/staking/interfaces/IStakingEvents.sol:8]: why streamShares and nVoteToken are indexed and account not?

File: contracts/dao/staking/interfaces/IStakingEvents.sol [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
     event StreamProposed(uint256 indexed streamId, address indexed streamOwner, address indexed rewardToken, uint256 maxDepositAmount);
     event Released(uint256 indexed streamId, address indexed user, uint256 pendingAmount);
     event StreamProposalCancelled(uint256 indexed streamId, address indexed owner, address indexed token);
     event StreamCreated(uint256 indexed streamId, address indexed owner, address indexed token, uint256 tokenAmount);
     event StreamRemoved(uint256 indexed streamId, address indexed owner, address indexed token);
     event Unstaked(address indexed account, uint256 indexed amount, uint256 indexed lockId);
+    event PartialUnstaked(address indexed account, uint256 indexed amount, uint256 indexed lockId);
```
Comment (BaldyAsh) [inline contracts/dao/staking/interfaces/IStakingEvents.sol:15]: why amount is indexed?

File: contracts/dao/staking/interfaces/IStakingEvents.sol [hunk 3]
```diff
@@ -5,12 +5,13 @@
...
 
 interface IStakingEvents {
-    event Staked(address account, uint256 indexed streamShares, uint256 indexed nVoteToken, uint256 indexed lockId);
+    event Staked(address account, uint256 amount,uint256 indexed streamShares, uint256 indexed nVoteToken, uint256 indexed lockId, uint256 end);
     event StreamProposed(uint256 indexed streamId, address indexed streamOwner, address indexed rewardToken, uint256 maxDepositAmount);
     event Released(uint256 indexed streamId, address indexed user, uint256 pendingAmount);
     event StreamProposalCancelled(uint256 indexed streamId, address indexed owner, address indexed token);
     event StreamCreated(uint256 indexed streamId, address indexed owner, address indexed token, uint256 tokenAmount);
     event StreamRemoved(uint256 indexed streamId, address indexed owner, address indexed token);
     event Unstaked(address indexed account, uint256 indexed amount, uint256 indexed lockId);
```
Comment (BaldyAsh) [inline contracts/dao/staking/interfaces/IStakingEvents.sol:14]: amount indexed?

File: contracts/dao/staking/interfaces/IStakingEvents.sol [hunk 4]
```diff
@@ -5,12 +5,13 @@
...
+    event PartialUnstaked(address indexed account, uint256 indexed amount, uint256 indexed lockId);
     event Pending(uint256 indexed streamId, address indexed account, uint256 indexed pendings);
```
Comment (BaldyAsh) [inline contracts/dao/staking/interfaces/IStakingEvents.sol:16]: pendings indexed?

**Review by BaldyAsh — CHANGES_REQUESTED** [review]
Pls give more clarity why amounts are indexed?

File: contracts/dao/staking/interfaces/IStakingEvents.sol (hunk 1 above)
Comment (ssubik) [reply contracts/dao/staking/interfaces/IStakingEvents.sol]: Removed indexed for steamShares and nVoteToken. Added indexed for account.

File: contracts/dao/staking/interfaces/IStakingEvents.sol (hunk 3 above)
Comment (ssubik) [reply contracts/dao/staking/interfaces/IStakingEvents.sol]: Removed index for amount

**Review by BaldyAsh — APPROVED** [review]

**Pull request merged**